src/components/About.js

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import { Container, Row, Col } from 'react-bootstrap';
+import { Card } from 'react-bootstrap';
+import Avatar from 'react-avatar';
```

Comment:
It looks like you have added a new package call "react-avatar" but there are no changes to the package.json file. If you want to use this package the package.json file also needs to be committed to this pull request.